Source/ACE.Database/Database.cs

```diff
@@ -66,8 +66,15 @@ public void AddPreparedStatement<T>(T id, params object[] parameters)
                             using (var command = new MySqlCommand(query.Item1.Query, connection, transaction))
                             {
                                 for (int i = 0; i < query.Item2.Length; i++)
+                                {
                                     command.Parameters.Add("", query.Item1.Types[i]).Value = query.Item2[i];
-
+#if NETWORKDEBUG
```

Comment:
This debug will output the values sent to the database in the console. I would like to change the flag from `NETWORKDEBUG` to `DBDEBUG`, or equivalent term.